commands/clone-api-plugins.js

```diff
@@ -0,0 +1,86 @@
+import { spawn } from "child_process";
+import fs from "fs";
+import inquirer from "inquirer";
+import checkIsProjectOfType from "../utils/checkIsProjectOfType.js";
+import apiPluginRepos from "../utils/apiPluginRepos.js";
+import Logger from "../utils/logger.js";
+
+/**
+ * @summary Creates api-plugins directory in it doesn't exist
+ * @return {void}
+ */
+function ensureApiPluginsDirectoryExists() {
+  const apiPluginsDir = `${process.cwd()}/api-plugins`;
+  if (!fs.existsSync(apiPluginsDir)) {
+    Logger.info("Creating api-plugins directory");
+    fs.mkdirSync(apiPluginsDir);
+  }
+}
+
+/**
+ * @summary Checks if plugin is already cloned
+ * @param {String} plugin - key of the plugin
+ * @return {boolean} - true if plugin is already cloned
+ */
+function checkIsPluginAlreadyCloned(plugin) {
```

Comment:
Generally if a function returns a boolean the name should be something like `isAlreadyCloned`

Reply:
I agree with that. I also will rename the utility function `checkIsProjectOfType`.